packages/xrpl/test/mockRippledTest.test.ts

```diff
@@ -27,12 +38,14 @@ describe('mock rippled tests', function () {
   })
 
   it('provide bad response shape in function', async function () {
-    this.mockRippled.suppressOutput = true
-    this.mockRippled.addResponse('account_info', (request) => {
+    if (testContext.mockRippled) {
```

Comment:
Why is this `if` used here but not for the rest of the test? Does `testContext.mockRippled!.suppressOutput = true` not work?

Reply:
Fixed